docs/installation.rst

```diff
@@ -0,0 +1,55 @@
+.. _installation:
+
+************
+Installation
+************
+
+Installing ``hvpy``
+-------------------
+
+If you are new to Python and/or do not have familiarity with `Python virtual environments <https://docs.python.org/3/tutorial/venv.html>`__, then we recommend starting by installing `miniforge <https://github.com/conda-forge/miniforge#miniforge3>`__.
+This works on all platforms (Linux, Mac, Windows) and installs a full-featured Python environment in your user directory without requiring root privileges.
```

Comment:
Have you worked with miniforge? I'm not comfortable recommending a package I haven't used.

Reply:
Yes, all the time.

Reply:
Sounds good. It looks like I'm going to have to use it too since I'm having issues with numpy on the Mac M1 chip.